jax/_src/array.py

```diff
@@ -680,54 +680,85 @@ def make_array_from_callback(
 def make_array_from_single_device_arrays(
     shape: Shape, sharding: Sharding, arrays: Sequence[basearray.Array]
 ) -> ArrayImpl:
-  r"""Returns a ``jax.Array`` from a sequence of ``jax.Array``\s on a single device.
+  r"""Returns a ``jax.Array`` from a sequence of ``jax.Array``\s each on an individual device.
```

Comment:
I think this sentence should be what it was before

Reply:
How about:
```
r"""Returns a ``jax.Array`` from a sequence of ``jax.Array``\s each on a single device.
```

Individual vs. single is a wordsmithing, I don't care about that. But pretty sure that "each" is fixing a real problem -- the sequence isn't on a single device but each participant in the sequnece is.

Reply:
keep each and single